src/_includes/_layouts/Home.jsx

```diff
@@ -113,6 +113,31 @@ export default ({ comp, title }) => {
             redirect_url={"projects"}
           />
         </section>
+        <section className="h-screen flex flex-col items-center justify-center py-16 gap-y-20 max-lg:h-fit ">
+          <h1 className="lg:text-5xl text-7xl">
+            <strong>How We Do It</strong>
+          </h1>
+          <div className=" grid grid-cols-2 gap-16 lg:px-40 px-20 max-lg:flex max-lg:flex-col place-items-center ">
```

Comment:
Could you increase the size of the image? In desktop is seems to be too small. Also, can you increase the top and bottom padding to increase the size of the Card.

Reply:
When I made the information card component, I hardcoded the height of the image to be around the same as the images in the Figma designs. I think they were consistently the same height for every image.